### PR TITLE
chore: pagination fixes

### PR DESCRIPTION
**Pull request by ice-hades**

### CODE REVIEW

**Review by ice-orion — DISMISSED** [review]

**Review by ice-hector — DISMISSED** [review]

File: src/store/modules/Referrals/actions/index.ts [hunk 1]
```diff
@@ -8,19 +8,19 @@ const GET_REFERRALS = ({referralType = 'T1'}: {referralType?: ReferralType}) =>
   createAction(
     'GET_REFERRALS',
     {
-      START: ({isInitial}: {isInitial?: boolean} = {}) => ({
+      START: ({isInitial = true}: {isInitial?: boolean} = {}) => ({
```
Comment (ice-atlas) [inline src/store/modules/Referrals/actions/index.ts]: Why we need `isInitial` as `true` by default? It is not obvious and easy to miss use it

Comment (ice-hades) [reply src/store/modules/Referrals/actions/index.ts]: for the pullToRefresh to work

Comment (ice-hades) [reply src/store/modules/Referrals/actions/index.ts]: `useRefresh`

Comment (ice-atlas) [reply src/store/modules/Referrals/actions/index.ts]: I got, but please let specify it as true explicitly there

Comment (ice-hades) [reply src/store/modules/Referrals/actions/index.ts]: fixed

**Review by ice-orion — APPROVED** [review]

**Review by ice-atlas — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Pull request merged**